lambdas/fetch-questions/src/types/questions-result-types.ts

```diff
@@ -0,0 +1,45 @@
+export class Info {
+  readonly taxYearCurrent: string | undefined;
+  readonly taxYearPrevious: string | undefined;
+
+  constructor(taxYearCurrent: string, taxYearPrevious: string) {
+    this.taxYearCurrent = taxYearCurrent;
+    this.taxYearPrevious = taxYearPrevious;
+  }
+}
+
+export class Question {
+  readonly questionKey: string;
+  readonly info: Info | undefined;
+
+  constructor(
+    questionKey: string,
+    taxYearCurrent: string | undefined,
+    taxYearPrevious: string | undefined
+  ) {
+    this.questionKey = questionKey;
+
+    this.info =
+      taxYearCurrent && taxYearPrevious
```

Comment:
Should there not be another case here for only currentTaxYear present?

Reply:
taxYearPrevious is now optional